### ISSUE TITLE
Zipper Context is Invalid for Deep-Select

### ISSUE THREAD

**Issue by dmlap** [djspiewak/anti-xml#22]
The code below produces java.lang.RuntimeException when entered into the scala console:

``` scala
import com.codecommit.antixml._
((<html><head><title></title></head><body /></html>).anti \\ "body").unselect
```

**Comment by djspiewak** [comment]
Yeah, I know about this one.  The zipper context is invalid for deep-select.  As you can imagine, it's a little tricky to make that one work.  Still trying to figure it out…

**Comment by djspiewak** [comment]
I've implemented a temporary fix in d62ada8f08879a7f4f8d494e835b6a094d9f69a8.  Basically, deep-select no longer returns a `Zipper`, but rather a `Group`.  I'd rather just make the zipper work for deep-select, but that's still an open problem.  Leaving this issue to track that effort.
